lib/service/desktop.go

```diff
@@ -253,6 +253,7 @@ func (process *TeleportProcess) initWindowsDesktopServiceRegistered(log *logrus.
 				"Windows desktop service %s:%s is starting on %v.",
 				teleport.Version, teleport.Gitref, listener.Addr())
 		}
+		process.BroadcastEvent(Event{Name: WindowsDesktopReady, Payload: nil})
```

Comment:
Before you ask, I will try to get an integration test running to exercise this, but wanted to get the fix out before the next set of releases.

We've desktop access integration tests so far because of the large number of dependencies to stand up desktop access (Active Directory, LDAP server, etc).

I think we can change that now since the service itself will start even if it cannot connect to LDAP.